assets/javascripts/kitten/components/navigation/header-nav/components/nav.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Item = ({ children, href, className, ...others }) => (
+  <li className="k-HeaderNav-nav__element" role="menuitem">
+    <a href={href} className={`k-HeaderNav-nav__item ${className}`} {...others}>
+      {children}
+    </a>
+  </li>
+)
+
+Item.proptypes = {
+  href: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+}
+
+export const Nav = ({ children }) => (
+  <nav className="k-HeaderNav-nav">
+    <ul role="menubar" className="k-HeaderNav-nav__list">
+      {children}
+    </ul>
+  </nav>
```

Comment:
```suggestion
  <div className="k-HeaderNav-nav">
    <ul className="k-HeaderNav-nav__list">
      {children}
    </ul>
  </div>
```

Si les styles de l'élément `k-HeaderNav-nav` peuvent être mergés dans `k-HeaderNav-nav__list` c'est tant mieux, ça éliminera une `div` inutile